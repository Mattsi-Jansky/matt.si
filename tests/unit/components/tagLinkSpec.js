import React from "react"
import renderer from "react-test-renderer"
import TagLink from "../../../src/components/tagLink"
import { shallow } from 'enzyme';

describe("TagLink", () => {
  it("hasn't changed", () => {
    const tagLink = renderer
      .create(<TagLink tag="test" />)
      .toJSON()
    expect(tagLink).toMatchSnapshot()
  })

  it("should render children", () => {
    const expectedText = "hello world"
    const tagLink = shallow(<TagLink tag="test">{expectedText}</TagLink>)

    expect(tagLink.text()).toEqual(expectedText)
  })
})
