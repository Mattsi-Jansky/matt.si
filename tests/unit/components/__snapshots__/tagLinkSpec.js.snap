// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagLink hasn't changed 1`] = `
<a
  href="/tags/test/"
  onClick={[Function]}
  onMouseEnter={[Function]}
/>
`;
